packages/wonder-blocks-modal/src/themes/default.ts

```diff
@@ -1,36 +1,69 @@
-import * as tokens from "@khanacademy/wonder-blocks-tokens";
+import {
+    border,
+    semanticColor,
+    spacing,
+} from "@khanacademy/wonder-blocks-tokens";
 
 const theme = {
-    color: {
-        bg: {
-            inverse: tokens.color.darkBlue,
+    /**
+     * Shared tokens
+     */
+    root: {
+        // TODO(WB-1852): Remove light variant.
+        color: {
+            inverse: {
```

Comment:
Is there a reason we don't need the background and foreground defined for the default case (non-inverse)?